rootex/script/interpreter.cpp

```diff
@@ -124,68 +124,44 @@ void LuaInterpreter::registerTypes()
 		sol::usertype<Vector2> vector2 = rootex.new_usertype<Vector2>(
 		    "Vector2",
 		    sol::constructors<Vector2(), Vector2(float, float)>(),
-		    sol::meta_function::addition, [](Vector2& l, Vector2& r)
-		    { return l + r; },
-		    sol::meta_function::subtraction, [](Vector2& l, Vector2& r)
-		    { return l - r; },
-		    sol::meta_function::multiplication, [](Vector2& l, Vector2& r)
-		    { return l * r; },
-		    sol::meta_function::multiplication, [](Vector2& l, float r)
-		    { return l * r; },
-		    sol::meta_function::division, [](Vector2& l, float r)
-		    { return l / r; },
-		    sol::meta_function::division, [](Vector2& l, Vector2& r)
-		    { return l / r; });
-		vector2["dot"] = [](const Vector2& l, const Vector2& r)
-		{ return l.Dot(r); };
-		vector2["cross"] = [](const Vector2& l, const Vector2& r)
-		{ return l.Cross(r); };
+		    sol::meta_function::addition, [](Vector2& l, Vector2& r) { return l + r; },
```

Comment:
Run a linter here to have this being the original, ideally your diff should only contain things which you've changed.